Gemfile

```diff
@@ -1,6 +1,7 @@
+# frozen_string_literal: true
+
 source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```

Comment:
I don't think you need this line unless you have Gem's coming from github sources.

```suggestion
```